eng/WpfArcadeSdk/tools/TestProjects.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+  <PropertyGroup Condition="'$(WpfTest)'=='true'">
+    <RuntimeIdentifier Condition="('$(Platform)'!='AnyCPU' and '$(Platform)'!='Win32') Or $(Platform.EndsWith('64'))">win-$(Platform)</RuntimeIdentifier>
+    <RuntimeIdentifier Condition="'$(Platform)'=='AnyCPU' or '$(Platform)'=='Win32' or '$(Platform)'=='x86'">win-x86</RuntimeIdentifier>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(IsTestProject)' == 'true' or '$(WpfTest)'=='true'">
+    <TestArchitectures Condition="'$(PlatformTarget)' == '' or '$(PlatformTarget)' == 'AnyCpu'">x86</TestArchitectures>
```

Comment:
Might be worth a comment here that Arcade handles x64 correct, but doesn't correlate AnyCpu to x86.  I saw you have one in configure-toolset.ps1.